### PR TITLE
#FN:54 chore: Create a README file

### PR DESCRIPTION
**Pull request by kunda4**
# PR Description
Add a detailed readme file for the API to include the information about Project
## Description of tasks that were expected to be completed
List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style
- [x] Create Readme file
- [x] Add Installation and run commands
- [x] Add Description
- [x] Add Dependencies
- [x] Add Tests setup and execution
- [x] Add Code Climate maintainability badge
- [x] Add Code Climate code coverage badge
- [x] Add CircleCI status badge
## Description of tasks that were expected to be completed and that are not yet completed
- [ ] Create .....
- [ ] Add .....
- [ ] ......
## How this may be tested?
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
1. run `yarn i` to install all required dependencies
2. run `yarn run dev`
3. step 3
4. step 4...
## Screenshots (If appropriate/applicable)
Provide screenshots related to your PR in case it's appropriate and applicable
## Please check this Checklist before you submit your PR:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by Lydie260 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,84 @@
-# ecommerce-app-cogito-fe
\ No newline at end of file
+bash```
```
Comment (dushimeemma) [inline README.md]: what does this mean?

File: README.md [hunk 2]
```diff
@@ -1 +1,84 @@
...
+Follow these steps to set up the cogito's Frontend codebase locally:
+
+Clone the repository:
+`git clone https://github.com/atlp-rwanda/ecommerce-app-cogito-fe.git`
+
+Go to the folder: 
+`cd ecommerce-app-cogito-fe`
+
+Install yarn Grobally in your local machine
+`npm install -g yarn`
```
Comment (dushimeemma) [inline README.md:56]: Do we need to run both commands at the same time?

File: README.md [hunk 3]
```diff
@@ -1 +1,84 @@
...
+
+Install dependencies:
+`yarn install`
+
+## Run The Service
+
+#### To run the cogito's Frontend service locally, follow these steps:
+
+Start the development server: 
+`yarn run dev`
```
Comment (dushimeemma) [inline README.md]: I think yarn dev is enough

File: README.md [hunk 4]
```diff
@@ -1 +1,84 @@
...
+Open your browser and visit http://127.0.0.1:5173/. to see the application running.
```
Comment (dushimeemma) [inline README.md]: Is this link always the same?

**Review by dushimeemma — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kunda4) [reply README.md]: I removed it

File: README.md (hunk 2 above)
Comment (kunda4) [reply README.md:56]: I choose to keep the first one as most known

File: README.md (hunk 4 above)
Comment (kunda4) [reply README.md]: I removed it

File: README.md [hunk 5]
```diff
@@ -1 +1,82 @@
...
+- Informative Reviews and Ratings: Provide and access insightful feedback and ratings for products.
+
+- notifications: Notify seller when the products is expired and when user's password needs to be updated
+-  public chat: enables real-time communication between users, facilitating product inquiries, customer support, issue resolution, and building trust with you. 
+
+## USING THE PROJECT
+
+### Technologies Used
+
+- Vite React
```
Comment (Illustre13) [inline README.md:36]: You can add all of these if you don't mind: 
React
Vite
Tailwind CSS
TypeScript
Redux Tool Kit

File: README.md [hunk 6]
```diff
@@ -1 +1,82 @@
...
+
+- Informative Reviews and Ratings: Provide and access insightful feedback and ratings for products.
+
+- notifications: Notify seller when the products is expired and when user's password needs to be updated
+-  public chat: enables real-time communication between users, facilitating product inquiries, customer support, issue resolution, and building trust with you. 
+
+## USING THE PROJECT
+
+### Technologies Used
+
```
Comment (Illustre13) [inline README.md:35]: Add 
TypeScript

**Review by Illustre13 — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (kunda4) [reply README.md:35]: done

**Review by Illustre13 — APPROVED** [review]

**Review by dushimeemma — APPROVED** [review]

**Pull request merged**